ci/azure-install-rust.yml

```diff
@@ -6,16 +6,18 @@ steps:
         rustup self update
       elif [ "$AGENT_OS" = "Windows_NT" ]; then
         curl -sSf -o rustup-init.exe https://win.rustup.rs
-        rustup-init.exe -y --default-toolchain $TOOLCHAIN
+        rustup-init.exe -y --default-toolchain $TOOLCHAIN --profile=minimal
```

Comment:
This works on all OSes
```suggestion
        curl -sSL https://sh.rustup.rs | sh -s -- -y --default-toolchain=$TOOLCHAIN --profile=minimal
```

Reply:
Sorry for the delay. I am now back from vacation. How do we best combine the branches?

Reply:
How about using a custom script: https://github.com/lzutao/junction/blob/ci-scripts/ci/install-rustup.bash ?